### PR TITLE
[stdlib][xml] Initial typestubs for `xml.dom`

### PR DESCRIPTION
**Pull request by jermenkoo**
Let's add some typestubs for `xml`. (as reported in #3787)

I added 2 only to get the feel and then I'll add more after the review.

### CODE REVIEW

File: stdlib/2and3/xml/dom/__init__.pyi [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+class UserDataHandler:
+    NODE_CLONED: int
+    NODE_IMPORTED: int
+    NODE_DELETED: int
+    NODE_RENAMED: int
+
+XML_NAMESPACE: str
+XMLNS_NAMESPACE: str
+XHTML_NAMESPACE: str
+EMPTY_NAMESPACE: NoneType
```
Comment (CraftSpider) [inline stdlib/2and3/xml/dom/__init__.pyi]: For this and the line below, you can type it as just `None`, this should fix the mypy errors

File: stdlib/2and3/xml/dom/__init__.pyi [hunk 2]
```diff
@@ -0,0 +1,67 @@
+class Node:
+    ELEMENT_NODE: int
```
Comment (CraftSpider) [inline stdlib/2and3/xml/dom/__init__.pyi]: For class-level variables, use `= ...`. Leave it off for module-level constants and class instance variables. For example, look at https://github.com/python/typeshed/blob/c50fce6ef3111bba4d0b46e3c1f8e846d05cf0c3/stdlib/2and3/bdb.pyi#L69

Comment (CraftSpider) [reply stdlib/2and3/xml/dom/__init__.pyi]: It's actually optional whether to have `= ...` for module-level constants, but from what I remember the encouraged style is to leave them off.

File: stdlib/2and3/xml/dom/__init__.pyi [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    ENTITY_NODE: int
+    PROCESSING_INSTRUCTION_NODE: int
+    COMMENT_NODE: int
+    DOCUMENT_NODE: int
+    DOCUMENT_TYPE_NODE: int
+    DOCUMENT_FRAGMENT_NODE: int
+    NOTATION_NODE: int
+
+
+#ExceptionCode
```
Comment (srittau) [inline stdlib/2and3/xml/dom/__init__.pyi]: Flake8 complains about the lack of a space behind the hash mark, the this is good to be merged.

**Comment by jermenkoo** [comment]
Thanks for the review, @srittau and @CraftSpider!

**Pull request merged**